app/views/grids/_people.html.erb

```diff
@@ -20,7 +20,7 @@
   </tr>
   <% if index == 9 %>
     <tr class="show-all">
-      <td><a>Show <%= users.length - 10 %> more <b class="caret"></b></a></td>
+      <td><a>Show <%= if(users.length>0)do users.length-10 + "more" else "less" end %>  <b class="caret"></b></a></td>
```

Comment:
`if` is followed by `do` here. I think you want to use `then` instead of `do`. Another option is to use the conditional operator.